### PR TITLE
[trivial] Add a link to the Bitcoin-Core repository and website to the About Dialog

### PR DESCRIPTION
**Pull request by maflcko**
Alternative to #8192

### CODE REVIEW

**Comment by btcdrak** [comment]
We should also have a link to the project website.

**Comment by gmaxwell** [comment]
Perhaps better to just link the website, in case we move off github in the future?

**Comment by btcdrak** [comment]
@gmaxwell agreed. The website address should be more permanent than the source repository which may or may not change in the future.

**Comment by MarcoFalke** [comment]
@btcdrak @gmaxwell I still think it is useful to have a link directly to the GitHub repo. I think there are no plans to move away from GitHub in the near future. Though, we may switch to the bitcoin-core organization some time...

Also, I have added two links to the website:

screenshot from 2016-06-15 12-48-57

**Comment by sipa** [comment]
utACK

**Comment by sipa** [comment]
I think listing both the source code and the project website is a good idea.

File: src/init.cpp [hunk 1]
```diff
@@ -479,11 +479,27 @@ std::string HelpMessage(HelpMessageMode mode)
...
     // todo: remove urls from translations on next change
     return CopyrightHolders(strprintf(_("Copyright (C) %i-%i"), 2009, COPYRIGHT_YEAR) + " ") + "\n" +
            "\n" +
            _("This is experimental software.") + "\n" +
            "\n" +
+           strprintf(_("The source code is available from %s."),
+               URL_SOURCE_CODE) +
+           "\n" +
+           "\n" +
+           strprintf(_("Additional information about %s is available at %s."),
```
Comment (paveljanik) [inline src/init.cpp]: Two %s can make the translation a bit hard in some languages where the order of words in the sentence is a bit different.

**Comment by btcdrak** [comment]
fine, but my personal preference would just link to "https://bitcoincore.org" rather than a specific page.

**Comment by paveljanik** [comment]
Yes, just one link please. It would be also nice to not have About windows scrollable because of lot of texts.

File: src/init.cpp (hunk 1 above)
Comment (MarcoFalke) [inline src/init.cpp]: :hankey:

Comment (MarcoFalke) [inline src/init.cpp]: Someone should write a macro that uses nested strprinf's, so you can use the number of %'s to determine the precedence of the format specifier

Comment (MarcoFalke) [inline src/init.cpp]: Thanks, worked around for now.

**Comment by MarcoFalke** [comment]
screenshot from 2016-06-15 14-12-02

**Comment by btcdrak** [comment]
utACK fa6565d

**Comment by paveljanik** [comment]
ACK https://github.com/bitcoin/bitcoin/commit/fa6565d2fc9c4cd8eac57915dc8b0aef8176b39c

**Comment by paveljanik** [comment]
When you are in this file, can you change the http:// to https:// in the MIT link?

**Comment by MarcoFalke** [comment]
@paveljanik  Please mention this in #8192 (I can't change already translated strings as part of this pull anymore, as we are freezing translations tomorrow)

**Comment by paveljanik** [comment]
@MarcoFalke ? You have just added the string "Please contribute if you..". So?

**Comment by MarcoFalke** [comment]
@paveljanik The problem is with already translated strings. Currently we have  161 project languages (27 without translators) and not even the German translations for 0.12 are fully done.

If you change a translated string, people would need to go through the about ~80 languages where it is already translated and apply the difference. This may or may not happen.

To sum up:
* Having no translation is better than having no link to our websites at all.
* Having a slightly outdated translation (e.g. a http link) is better than no translation at all.

**Comment by paveljanik** [comment]
Sure, but in the case of http -> https, we can simply postprocess the transifex files.

**Comment by jonasschnelli** [comment]
Please squash.
ACK fa6565d2fc9c4cd8eac57915dc8b0aef8176b39c

**Comment by MarcoFalke** [comment]
@jonasschnelli: Squashed fa6565d into fa58e5e

**Comment by laanwj** [comment]
> Perhaps better to just link the website, in case we move off github in the future?

In that case we'll just update the software.

**Pull request merged**